Website/Providers/DataProviders/SqlDataProvider/09.03.00.SqlDataProvider

```diff
@@ -378,6 +378,23 @@ CREATE PROCEDURE {databaseOwner}[{objectQualifier}Journal_ListForGroup]
 	ORDER BY RowNumber ASC;
 GO
 
+IF EXISTS (SELECT * FROM dbo.sysobjects
```

Comment:
I'd Omid this check - if HostSettings table is missing, the script should fail, because there is a serious problem.